ui-tests/src/test/java/org/openmrs/reference/page/ConditionsPage.java

```diff
@@ -18,6 +18,7 @@
     private static final By EDIT = By.cssSelector("i[title='Edit Condition: ']");
     private static final By DELETE = By.cssSelector("i[title='Delete']");
     private static final By CONDITIONS_LIST = By.cssSelector("tr.clickable-tr");
+    private static final By CONFIRM_DELETE_CONDITION = By.cssSelector("#remove-condition-dialog > div.dialog-content > button.confirm.right");
```

Comment:
Is this a link, label of what?

Reply:
> Is this a link, label or what?

@dkayiwa Its a `Button`  but unfournately it has no id

Reply:
> Is this a link, label or what?

@dkayiwa Its a `Button`  but unfournately it has no id

Reply:
How do i tell so from its name?

Reply:
> How do i tell so from its name?

@dkayiwa should it be something like this `CONFIRM_DELETE_CONDITION_BUTTON`

Reply:
> How do i tell so from its name?

@dkayiwa  done